### PR TITLE
adding user personas

### PR DESCRIPTION
**Pull request by Mervatabuamro**
Adding user personas documentation

### CODE REVIEW

**Review by jialvaradob — APPROVED** [review]

**Comment by jialvaradob** [comment]
Good job! Thanks for linking the issue as well  💯

**Review by jialvaradob — CHANGES_REQUESTED** [review]
The newly created file is empty, check saving changes on your vscode or maybe paste it directly in github as an edit

**Review by Ibrahim86Ali — APPROVED** [review]

**Comment by Mervatabuamro** [comment]
OH I did not notice that
now its ready

**Review by jialvaradob — APPROVED** [review]

File: planning/user-personas.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# User Personas
+
+Our Sound Blog Platform is for many persons including
+
+----
+
+## Lexy/ Housewife 
+
+- **Bio**: Lexy is a housewife, 35 years old. Most of her day is taking care of kids and house chores. And meeting with neighbors.
```
Comment (Neha-2012) [inline planning/user-personas.md:9]: Spelling Mistake.

**Review by Neha-2012 — APPROVED** [review]

**Pull request merged**